internal/project/pkgfile/build.go

```diff
@@ -201,7 +201,7 @@ func (pkgfile *Build) CompileGitHubWorkflow(output *ghworkflow.Output) error {
 		Uses: "docker/login-action@" + config.LoginActionVersion,
 		With: map[string]string{
 			"registry": "ghcr.io",
-			"username": "${{ github.repository_owner }}",
+			"username": "${GITHUB_REPOSITORY_OWNER@L}",
```

Comment:
`@L`, what's that?, though I think this should not be needed for registry auth

Reply:
https://github.com/orgs/community/discussions/25768#discussioncomment-8057564

It's a bash only way to lowercase strings

Reply:
I've tested without, it works. I'm removing it